<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowsClass</class>
 <widget class="QMainWindow" name="MainWindowsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindows</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionPainter_2_0"/>
    <addaction name="actionProgrammer"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actReturn"/>
   <addaction name="actRotate"/>
   <addaction name="actShear"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actEraser"/>
   <addaction name="actPen"/>
   <addaction name="actRect"/>
   <addaction name="actEllipse"/>
   <addaction name="separator"/>
   <addaction name="actfill4"/>
   <addaction name="actFillScan"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>90</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QComboBox" name="comboBoxLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>实线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>点线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>破折号线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>点破折号线</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>黑色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>白色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>红色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>绿色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蓝色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>粉色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>紫红色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>黄色</string>
      </property>
     </item>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>线形</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>笔粗</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1到25之间</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_data">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>数据</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/open_pokeball_24px_1203093_easyicon.net.png</normaloff>:/MainWindows/Resources/open_pokeball_24px_1203093_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/abra_26.608695652174px_1203038_easyicon.net.png</normaloff>:/MainWindows/Resources/abra_26.608695652174px_1203038_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/bellsprout_19.235294117647px_1203045_easyicon.net.png</normaloff>:/MainWindows/Resources/bellsprout_19.235294117647px_1203045_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/charmander_21.021897810219px_1203052_easyicon.net.png</normaloff>:/MainWindows/Resources/charmander_21.021897810219px_1203052_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actReturn">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/eevee_20.029197080292px_1203058_easyicon.net.png</normaloff>:/MainWindows/Resources/eevee_20.029197080292px_1203058_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
  </action>
  <action name="actRotate">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/dratini_24.058536585366px_1203057_easyicon.net.png</normaloff>:/MainWindows/Resources/dratini_24.058536585366px_1203057_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="actShear">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/blue_team_16.759124087591px_1203046_easyicon.net.png</normaloff>:/MainWindows/Resources/blue_team_16.759124087591px_1203046_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>拉伸</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/camera_21.529411764706px_1203049_easyicon.net.png</normaloff>:/MainWindows/Resources/camera_21.529411764706px_1203049_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actEraser">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/mew_31.587096774194px_1203088_easyicon.net.png</normaloff>:/MainWindows/Resources/mew_31.587096774194px_1203088_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>橡皮</string>
   </property>
  </action>
  <action name="actPen">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/squirtle_19.153284671533px_1203122_easyicon.net.png</normaloff>:/MainWindows/Resources/squirtle_19.153284671533px_1203122_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>笔</string>
   </property>
  </action>
  <action name="actRect">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/snorlax_28.182857142857px_1203121_easyicon.net.png</normaloff>:/MainWindows/Resources/snorlax_28.182857142857px_1203121_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="actEllipse">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/pikachu_25.554404145078px_1203098_easyicon.net.png</normaloff>:/MainWindows/Resources/pikachu_25.554404145078px_1203098_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
  </action>
  <action name="actionPainter_2_0">
   <property name="text">
    <string>painter 2.0</string>
   </property>
  </action>
  <action name="actionProgrammer">
   <property name="text">
    <string>programmer</string>
   </property>
  </action>
  <action name="actfill4">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/psyduck_19.473945409429px_1203114_easyicon.net.png</normaloff>:/MainWindows/Resources/psyduck_19.473945409429px_1203114_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>四邻域填充</string>
   </property>
  </action>
  <action name="actFillScan">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindows/Resources/eevee_20.029197080292px_1203058_easyicon.net.png</normaloff>:/MainWindows/Resources/eevee_20.029197080292px_1203058_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>扫描填充</string>
   </property>
   <property name="toolTip">
    <string>扫描填充</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindows.qrc"/>
 </resources>
 <connections/>
</ui>
